### ISSUE TITLE
Android user cannot edit start and stop time of a time entry 

### ISSUE THREAD

**Issue by maritoggl** [toggl/mobileapp#3504]
## ⛑️ User Support
User creates a time entry and tries to edit its start and stop time. Changes do not get applied. 
Screencast available in conversations/19101048643. Before you ask - yes, the user did confirm changes by hitting the ok button :)

## About the user

App version: Giskard 1.5.2
Phone OS: Android 5.0
Phone model: Galaxy Note 3?

## Expected behaviour
Changes get saved when changes are confirmed. 

## To Reproduce
See the screencast linked above.

**Comment by epagnini** [comment]
conversations/19222498704

**Comment by maritoggl** [comment]
conversations/19222737538

**Comment by nicktoggl** [comment]
conversations/19086215415

**Comment by amulware** [comment]
What's the status of this?

**Comment by kornelijepetak** [comment]
Status: Happens only on Android 5. Unfortunately, the 🍭  behaves differently in this regard and as far as I am aware, we need to add some custom logic only for 🍭 . With all the 🍭-specific changes, the code is getting messier... 😕

**Comment by maritoggl** [comment]
For tracking purposes, [link to another report](https://app.intercom.io/a/apps/ayixs927/inbox/inbox/unassigned/conversations/19786921936)

**Comment by daividssilverio** [comment]
About this issue, as @kornelijepetak pointed out, 🍭 doesn't work as the callbacks are not called when the times are picked in the time picker.

There is one option that I've tested that is to use stock spinner time picker that does emit the callbacks when the time changes on the picker, but it doesn't just work with our current implementation, it will require some tweaking and adaptation to use the spinner only for 🍭 and the radial picker otherwise.

It would could work even better if we used the spinner for the time picker and and the date picker both on 🍭 , since the DatePicker also doesn't work well there.
